core/src/components/progress-bar/test/shape/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Progress Bar - Shape</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <link href="../../../../../css/ionic.bundle.css" rel="stylesheet" />
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet" />
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script nomodule src="../../../../../dist/ionic/ionic.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+
+    <style>
+      ion-content::part(scroll) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+      }
+
+      ion-content ion-progress-bar {
+        margin: 10px 0;
+        width: 70%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <ion-app>
+      <ion-header>
+        <ion-toolbar>
+          <ion-title>Progress Bar - Shape</ion-title>
+        </ion-toolbar>
+      </ion-header>
+
+      <ion-content class="outer-content">
+        <ion-label> Default </ion-label>
```

Comment:
Let's match the other test pages and also use list header.

```suggestion
    <ion-list-header>
        <ion-label> Default </ion-label>
     </ion-list-header>
```
Do the same for the rest of the `ion-label`.

Reply:
The reason why I didn't use an ion-list is because of this rule:
![image](https://github.com/user-attachments/assets/284c7d31-8755-40e5-b84e-0f566a8b71dc)
It is overriding the progress-bar-shape rules.

Reply:
Good catch, then let's use `ion-item` and `lines="none"`

```html
<ion-list lines="none">
  <ion-list-header>
    <ion-label> Default </ion-label>
  </ion-list-header>
  <ion-item>
    <ion-progress-bar value="0.50"></ion-progress-bar>
  </ion-item>

  <ion-list-header>
    <ion-label> Round </ion-label>
  </ion-list-header>
  <ion-item>
    <ion-progress-bar value="0.50" shape="round"></ion-progress-bar>
  </ion-item>

  <ion-list-header>
    <ion-label> Rectangular </ion-label>
  </ion-list-header>
  <ion-item>
    <ion-progress-bar value="0.50" shape="rectangular"></ion-progress-bar>
  </ion-item>
</ion-list>
```